app/init/resources.php

```diff
@@ -947,6 +974,37 @@ function getDevice(string $root, string $connection = ''): Device
     return Key::decode($project, $key);
 }, ['request', 'project']);
 
+
+App::setResource('store', function (): Store {
+    return new Store();
+});
+
+App::setResource('proofForPassword', function (): Password {
+    $hash = new Argon2();
+    $hash
+        ->setMemoryCost(7168)
+        ->setTimeCost(5)
+        ->setThreads(1);
```

Comment:
There are a couple of other isntances of Argon2, lets set the same values for all